src/grid/Row/index.tsx

```diff
@@ -16,7 +16,14 @@ const GridRow = ({ index }: GridRowProps) => {
 
   const yPos = useMemo(() => rowHeight * (index + 1), [index, rowHeight]);
 
-  return <KonvaLine points={[drawRangeStart, yPos, drawRangeEnd, yPos]} stroke={themeColor} />;
+  const fill = useMemo(() => (index % 2 === 0 ? "#F0F0F0" : "rgb(255,255,255)"), [index]);
```

Comment:
Create some constants in theme file for DEFAULT_ROW and ALTERNATE_ROW.
Test also in dark mode to see if they are okay